app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -132,22 +141,95 @@ class ReportingRepositoryImpl(
         tanUuid = apiInteractor.requestTan(mobileNumber).uuid
     }
 
-    override suspend fun uploadReportInformation(
-        temporaryExposureKeys: List<TemporaryExposureKey>
-    ): MessageType {
+    override suspend fun uploadReportInformation(): MessageType {
         return when (messageTypeSubject.value) {
-            MessageType.Revoke.Suspicion -> uploadRevokeSuspicionInfo(temporaryExposureKeys)
-            MessageType.Revoke.Sickness -> uploadRevokeSicknessInfo(temporaryExposureKeys)
-            else -> uploadInfectionInfo(temporaryExposureKeys)
+            MessageType.Revoke.Suspicion -> uploadRevokeSuspicionInfo()
```

Comment:
Do we need this RX indirection here? we not just hand over the `reportedInfectionLevel`???

Reply:
Sorry, I'm not sure to which RX indirection are you referring in this case. The only update for this method was the removal of temporary exposure keys as parameter for the methods.

Reply:
no rx there, let's keep it.